source/features/useful-forks.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'dom-chef';
+import elementReady from 'element-ready';
+import * as pageDetect from 'github-url-detection';
+import {RepoForkedIcon} from '@primer/octicons-react';
+
+import features from '.';
+import {getRepo} from '../github-helpers';
+import looseParseInt from '../helpers/loose-parse-int';
+
+async function init(): Promise<void | false> {
+	const forkCountBadgeValue = await elementReady('.social-count[href$="/network/members"]');
+	const forkCount = looseParseInt(forkCountBadgeValue!);
+	if (forkCount === 0) {
```

Comment:
```suggestion
	const forkCount = await elementReady('.social-count[href$="/network/members"]')!;
	if (looseParseInt(forkCount) === 0) {
```

Reply:
Would've caused an error. I pushed a fix with aligns with this suggestion.